v2/internal/runner/options.go

```diff
@@ -195,9 +186,65 @@ func validateOptions(options *types.Options) error {
 		return errors.New("ipv4 and/or ipv6 must be selected")
 	}
 
+	// Validate cloud option
+	if err := validateCloudOptions(options); err != nil {
+		return err
+	}
 	return nil
 }
 
+func validateCloudOptions(options *types.Options) error {
+	if options.ScanList || options.DeleteScan != "" || options.ScanOutput != "" || options.ListDatasources || options.ListTargets || options.ListTemplates || options.RemoveDatasource != "" || options.AddTarget != "" || options.AddTemplate != "" || options.RemoveTarget != "" || options.RemoveTemplate != "" || options.GetTarget != "" || options.GetTemplate != "" {
```

Comment:
For better readability, I'd suggest refactoring this as a helper function of `options`:
```go
func (o *options) HasCloudOptions() bool {
    return options.ScanList || ...
}
```